lib/tasks/spree_mailchipm_ecommerce_tasks.rake

```diff
@@ -9,4 +9,16 @@ namespace :spree_mailchimp_ecommerce do
                                 currency_code: "USD"
                               })
   end
+
+  desc "Initial uploading of products list"
+  task upload_products: :environment do
+    logger = Logger.new("log/spree_mailchimp_initial_upload.log")
+    Spree::Product.find_each do |product|
+      begin
+        SpreeMailchimpEcommerce::CreateOrderCartJob.perform_now(product.id)
+      rescue StandardError
+        logger.error(product)
```

Comment:
we want to have more descriptive message here. Like product `id`, error message etc.